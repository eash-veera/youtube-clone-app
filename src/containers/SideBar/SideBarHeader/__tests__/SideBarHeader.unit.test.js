import React from 'react';
import { shallow } from 'enzyme';
import { SideBarHeader } from '../SideBarHeader';


describe('SideBarHeader', () => {

    test('Renders sidebar header with props.title=null', () => {
        const wrapper = shallow(
            <SideBarHeader/>
        );
        expect(wrapper).toMatchSnapshot();
    });

    test('Renders sidebar header with props.title=\'\'', () => {
        const wrapper = shallow(
            <SideBarHeader title=''/>
        );
        expect(wrapper).toMatchSnapshot();
    });

    test('Renders sidebar header with props.title="Sample Title"', () => {
        const wrapper = shallow(
            <SideBarHeader title='Sample Title'/>
        );
        expect(wrapper).toMatchSnapshot();
    });

}); 

