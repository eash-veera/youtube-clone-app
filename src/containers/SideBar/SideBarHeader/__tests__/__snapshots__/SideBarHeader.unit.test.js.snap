// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideBarHeader Renders sidebar header with props.title="Sample Title" 1`] = `
<MenuItem>
  <MenuHeader
    className="side-bar-header"
  >
    SAMPLE TITLE
  </MenuHeader>
</MenuItem>
`;

exports[`SideBarHeader Renders sidebar header with props.title='' 1`] = `
<MenuItem>
  <MenuHeader
    className="side-bar-header"
  />
</MenuItem>
`;

exports[`SideBarHeader Renders sidebar header with props.title=null 1`] = `
<MenuItem>
  <MenuHeader
    className="side-bar-header"
  />
</MenuItem>
`;
